src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentsResource.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentsResource : ResourceBase, ITrustlyPaymentsResource
+    {
+        public TrustlyPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<TrustlyPayment> Create(TrustlyPaymentRequest paymentRequest)
+        {
+            return Create(paymentRequest, PaymentExpand.None);
+        }
+
+        public Task<TrustlyPayment> Create(TrustlyPaymentRequest paymentRequest, PaymentExpand paymentExpand)
+        {
+            return TrustlyPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
+        }
+
+        public Task<TrustlyPayment> Get(Uri id)
```

Comment:
Codacy found an issue: [Make 'Get' a static method.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)

Reply:
@asbjornu 
This method can't be static, since it calls the other Get method.
Then the other Get method has to be static and that one uses this.httpClient that hasn't been initiated if the method would be static.